cepheus-cep/pom.xml

```diff
@@ -100,6 +100,11 @@
             <groupId>javax.inject</groupId>
             <artifactId>javax.inject</artifactId>
         </dependency>
+        <dependency>
+	    <groupId>commons-lang</groupId>
```

Comment:
Why are you introducing this dependency ? Commit does not make this clear...

Reply:
I have import "org.apache.commons.lang.StringUtils" in SubscriptionManager.java file to check the null value in service name and service path string.
code: 
if (StringUtils.isNotEmpty(provider.getServiceName())|| StringUtils.isNotEmpty(provider.getServicePath())) {
 			httpHeaders = getHeadersForProvider(provider);
}